macOS/DuckDuckGo/NetworkProtection/AppTargets/DeveloperIDTarget/VPNSubscriptionEventsHandler.swift

```diff
@@ -97,76 +93,102 @@ final class VPNSubscriptionEventsHandler {
                     }
 
                     let hasEntitlements = payload.entitlements.contains(.networkProtection)
-                    await self.handleEntitlementsChange(hasEntitlements: hasEntitlements, trigger: .notification(sourceObject: notification.object))
+                    await self.handleEntitlementsChangeNotification(hasEntitlements: hasEntitlements, sourceObject: notification.object)
                 }
             }
             .store(in: &cancellables)
     }
 
+    private func performClientCheck(trigger: VPNSubscriptionClientCheckPixel.Trigger) {
+        Task {
+            do {
+                let hasEntitlement = try await subscriptionManager.isFeatureEnabled(.networkProtection)
```

Comment:
This was changed and is a big thing in this PR: changed from `try?` to `try`.  This call can fail due to transient networking errors, etc.  If there's any error that requires the subscription to be removed, then this method should not be failing (ie: in fact I think it's not) but instead return `false`... so the `try?` was really bad.